spec/features/visitor/sign_out_spec.rb

```diff
@@ -1,13 +1,13 @@
 require "rails_helper"
 
-feature "Sign out" do
+feature "Sign Out" do
```

Comment:
I believe sign out spec should moved to the `user` folder.

Here are conventions we could apply:
* user - authenticated user
* visitor - anon user